src/periodograms.jl

```diff
@@ -357,7 +358,7 @@ function spectrogram(s::AbstractVector{T}, n::Int=length(s)>>3, noverlap::Int=n>
 
     out = stft(s, n, noverlap, PSDOnly(); onesided=onesided, nfft=nfft, fs=fs, window=window)
     Spectrogram(out, onesided ? rfftfreq(nfft, fs) : fftfreq(nfft, fs),
-                ((0:size(out,2)-1)*(n-noverlap)+n/2)/fs)
+                (n/2 : n-noverlap : (size(out,2)-1)*(n-noverlap)+n/2) / fs)
```

Comment:
`range + scalar` is deprecated in 0.7, `range .+ scalar` produces a `Vector` in 0.6, so this rewrite seemed most straight-forward for cross-version support.